### ISSUE TITLE
XCode 7.1 "include of non-modular header inside module 'Ono.Ono'"

### ISSUE THREAD

**Issue by dcunited001** [mattt/Ono#45]
I'm working on a pod called [Spectra](https://github.com/dcunited001/Spectra) that uses Ono 1.2.2 to parse XML for a scene graph and some other things.

I upgraded to XCode 7.1 yesterday and i haven't been able to resolve this build error at all.  I've found a lot of people online asking about this, but for the most part, they're not resolving this issue for a pod their building.  So many of the solutions i've found don't work for me.

```
include of non-modular header inside framework module Ono.Ono
```

I've found that by changing the following line in `ono.h` resolve the Ono build issues for me.

```c++
// changed from this
#import <Ono/ONOXMLDocument.h>

// to this
#import "ONOXMLDocument.h"
```

**Comment by cezheng** [comment]
@dcunited001  Hello, I guess you could try opening a pull request for this but mattt seems to be inactive on github for 5 months already, so I guess relying on your private fork of Ono should be better.

Also, since I have found & fixed a few bugs I met when using Ono several months ago and couldn't get the pull request merged, I finally decided to create my own port on Ono to Swift. I found that your Pod is using Swift so maybe you can try this: https://github.com/cezheng/Fuzi

**Comment by master-nevi** [comment]
This is not an issue with Ono, it's a CocoaPods bug when using `use_frameworks!` option in your Podfile. This has been fixed on the master branch of CocoaPods so I'm guessing the fix will be included in v0.39.1 or v0.40.0.

See this thread for more info: https://github.com/CocoaPods/CocoaPods/issues/4420

**Comment by dcunited001** [comment]
i believe i was using `carthage` here and running into problems.  

or that seems incorrect.  rather, i have since switched to carthage.  but i was still running into problems with `Ono`, so i switched to the swift `Ono` implementation called `Fuzi`
